core/frontend/src/tile/GltfReader.ts

```diff
@@ -1340,9 +1340,9 @@ export abstract class GltfReader {
           max = ext.decodedMax;
         } else {
           // Assume KHR_mesh_quantization...
-          min = posView.accessor.min;
-          max = posView.accessor.max;
-        }
+          min = [0, 0, 0];
+          max = [65535, 65535, 65535];
```

Comment:
where doess this value come from? should this be a constant, or should there be a comment for this?

Reply:
It's just the maximum value of an unsigned short.

Reply:
Ahh, I didn't see that that could be an unsigned byte as well.  I'll push a fix.

Reply:
Fix pushed.  Or is there some defined constant for these values that I should use?

Reply:
OK, I changed them to 0xFFFF and 0xFF which is what I see being used around in other places.